firewood/src/merkle/node.rs

```diff
@@ -427,6 +431,7 @@ impl Storable for Node {
     }
 
     fn serialize(&self, to: &mut [u8]) -> Result<(), ShaleError> {
+        debug!("[{self:p}] Serializing node");
```

Comment:
Forgot a feature flag here. This shouldn't happen if you follow my comment from above